telmos_goods.py

```diff
@@ -35,26 +35,24 @@ def load_goods_data(goods_file, hgv_output, lgv_output):
     hgv_df = pd.DataFrame(hgv_lines, columns=["I", "J", "V"], dtype="float64")
     lgv_df = pd.DataFrame(lgv_lines, columns=["I", "J", "V"], dtype="float64")
     
-    assert len(hgv_df) == 803*803, "HGV file requires 803 * 803 entries"
-    assert len(lgv_df) == 803*803, "LGV file requires 803 * 803 entries"
+    if not len(hgv_df) == 803*803:
+        raise Exception("HGV file requires 803 * 803 entries")
+    if not len(lgv_df) == 803*803:
+        raise Exception("LGV file requires 803 * 803 entries")

```

Comment:
```suggestion
    if not len(hgv_df) == 803*803:
        raise ValueError("HGV file requires 803 * 803 entries")
    if not len(lgv_df) == 803*803:
        raise ValueError("LGV file requires 803 * 803 entries")
```

Generic `Exception` calls aren't ideal - I'd view this as an error with the values being passed through, thus `ValueError`?